packages/expandable_reorderable_list/pubspec.yaml

```diff
@@ -3,7 +3,7 @@ description: A wrapper around ReorderableListView that allows you to expand and
 repository: https://github.com/Novade/flutter_packages/tree/master/packages/expandable_reorderable_list
 issue_tracker: https://github.com/Novade/flutter_packages/issues
 
-version: 1.0.0
+version: 1.0.1
```

Comment:
The scroll direction is a new feature so this should be 1.1.0

Reply:
I fixed this in https://github.com/Novade/flutter_packages/pull/18/commits/8396f348ca7f014bad5381626ec178ed6fc8bcd2